app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_post_comments_counter
+
+  private
+
+  def update_post_comments_counter
+    Post.increment_counter(:comments_counter, post_id)
+  end
```

Comment:
- [ ] Please, kindly take note that I was not able to access this private method with `rails c` command. The reason is because Ruby is case sensitive, it seems like you're calling `Post model` instead of `post` parameter declared above.  I will suggest that you increment the value of the `post` parameter above that is referring to current instance of the Post model.

ℹ️ Note that: 
1. The association indicated above allows you to navigate between the `Comment and Post` models and retrieve the associated records. 🙏

2. You can as well implement it as you did, but kindly remove the **`private`** command so that `Post` model can be publicly accessible by `Comment` class. 🙏 

**Kindly make necessary corrections for the counters in `like` and `post` models as well** 🙏